python/common/org/python/types/Object.java

```diff
@@ -979,4 +990,110 @@ public void __delitem__(org.python.Object index) {
     public org.python.Object __round__(org.python.Object ndigits) {
         throw new org.python.exceptions.AttributeError(this, "__round__");
     }
+
+    // Need to implement Is __eq__ Is Not __ne__
+    // In __contains__ (reversed operands) Not In __not_contains__ (reversed operands)
+    //
+    // for Is and Is Not, falls through to here only when either side is a constant in python
+    // which ends up as org.python.types.[Int|Float|Complex], otherwise it's dealt with
+    // by object reference comparison IF_ACMEQ. so we fall through to __eq__!
+    public enum CMP_OP {
+        GE(">=", "__ge__", "__le__"),
```

Comment:
Is this right? Shouldn't the reflection of `__ge__` be `__lt__`, rather than `__le__` (similarly for the other operations)? And isn't is the reflection of `__eq__` `__ne__` (and vice versa)? Or am I missing what the reflection is used for here?